Sprint-3/reading-list/script.js

```diff
@@ -21,3 +22,30 @@ const books = [
   },
 ];
 
+function readingList(books) {
+  const list = document.getElementById("reading-list");
+
+  books.forEach((book) => {
+    const listItem = document.createElement("li");
+    listItem.className = "book-item";
+    listItem.style.backgroundColor = book.alreadyRead ? "#c8f7c5" : "#f7c5c5";
+
+    const title = document.createElement("h3");
+    title.textContent = book.title;
+
+    const author = document.createElement("p");
+    author.textContent = `by ${book.author}`;
+
+    const image = document.createElement("img");
+    image.src = book.bookCoverImage;
+    image.alt = `${book.title} cover`;
+    image.style.width = "100px";
```

Comment:
Here is another place where using a class name (for styling) is preferred to styling the image within the javascript code.